composer.json

```diff
@@ -509,6 +510,16 @@
 			"approved-by": "JF",
 			"approved-date": "2021-11-21"
 		},
+		"league/commonmark": {
+			"source": "https://github.com/thephpleague/commonmark",
+			"used_version": "2.3.9",
+			"added_by": "Thibeau Fuhrer <thibeau@sr.solutions>",
+			"wrapped_by": "ILIAS\\UI\\Component\\Input\\Field\\MarkdownRenderer",
+			"last_update": "2023-03-14",
+			"last_update_by": "Thibeau Fuhrer <thibeau@sr.solutions>",
+			"approved-by": "JF",
+			"approved-date": "2021-11-21"
```

Comment:
@thibsy I guess this date is a copy-paste typo ;)

Reply:
thx @tfamula, you have a sharp eye ;) I fixed this with https://github.com/ILIAS-eLearning/ILIAS/commit/6a9aaed4ce16830cc6fab484cf73a6695d7cd8bf